google-api-client/src/main/java/com/google/api/client/googleapis/json/GoogleJsonResponseException.java

```diff
@@ -141,6 +141,9 @@ public static GoogleJsonResponseException from(JsonFactory jsonFactory, HttpResp
     }
     // message
     StringBuilder message = HttpResponseException.computeMessageBuffer(response);
+    (message.length() == 0 ? message : message.append(StringUtils.LINE_SEPARATOR))
```

Comment:
It took me some effort to decode this. It would be clearer as a simple if clause with no ternary operator.

Reply:
Done.